_episodes/01-select.md

```diff
@@ -15,18 +15,19 @@ keypoints:
 - "Use SELECT... FROM... to get values from a database table."
 - "SQL is case-insensitive (but data is case-sensitive)."
 ---
-A [relational database]({{ site.github.url }}/reference.html#relational-database)
+A [relational database]({{ page.root }}/reference/#relational-database).
```

Comment:
Using `/reference/` or `/setup/` is going to bring back #252

Reply:
I think #252 is not resolved. The links are failing for me to get to the right definitions.

Reply:
Do you have an example? I checked all the anchors with the W3C link checker.